syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTBasketItem";
option optimize_for = LITE_RUNTIME;

message BasketItem
{
    optional uint32 version = 1;
    optional uint64 weight = 2;
    optional string unit = 3;
    optional string account = 4;
}
